src/virtual-wan/HISTORY.rst

```diff
@@ -3,6 +3,17 @@
 Release History
 ===============
 
+0.2.16
+++++++
+* `az network vhub`: Add new subgroup `route-map` to support managing virtual hub route map.
+* Bump api version from 2021-08-01 to 2022-07-01.
```

Comment:
```suggestion
```

Reply:
`az network vhub connection`, `az network p2s-vpn-gateway` and `network vpn-gateway` have not migrated to codegen v2, so the new parameters(`associated_inbound_routemap` and `associated_outbound_routemap`) use the new sdk.

Reply:
IMO, @jsntcy's point is API version should be transparent to the users.